test/rendition.postprocess.test.js

```diff
@@ -25,9 +25,9 @@ const fs = require('fs-extra');
 const { MetricsTestHelper } = require("@adobe/asset-compute-commons");
 
 const PNG_FILE = "test/files/fileSmall.png";
-const BMP_FILE = "test/files/file.bmp";
-const GIF_FILE = "test/files/file.gif";
-const JPG_FILE = "test/files/file.jpg";
+// const BMP_FILE = "test/files/file.bmp";
```

Comment:
Instead of commenting, why not remove it?

Reply:
These were going to be used in the tests when the test is completed. for now I commented it out to ignore the eslint changes